packages/location-field/src/stories/Desktop.story.js

```diff
@@ -1,14 +1,18 @@
 import React from "react";
 
+import { Ship } from "styled-icons/fa-solid";
 import LocationField from "..";
 import {
   currentPosition,
   geocoderConfig,
   getCurrentPosition,
   onLocationSelected,
-  selectedLocation
+  selectedLocation,
+  layerColorMap
```

Comment:
Sort names in this import.

Reply:
Sorted in e0fc8a811358ed8a55a587f948db1614da68dbd4